eventlet/patcher.py

```diff
@@ -312,6 +312,26 @@ def monkey_patch(**on):
             for attr_name in deleted:
                 if hasattr(orig_mod, attr_name):
                     delattr(orig_mod, attr_name)
+
+            if name == 'threading' and sys.version_info >= (3, 7):
```

Comment:
Or maybe this should say `... and hasattr(_os, 'register_at_fork'):`?

Reply:
i think yes, they may remove register_at_fork in future version; checking py version in test is the right move though